_posts/2021-11-19-migrating-kafka-with-mirror-maker2.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+title: "Migrating Kafka clusters with MirrorMaker2 and Strimzi"
+date: 2021-11-09
+author: roman_martin,manuel_schindler
```

Comment:
Will it be a blocker to publish the blog? If so, we could remove one of the authors (e.g. myself) when you accepted this blog.

Reply:
I'm sure it is not a blocker from our side. It is more that we want to give you the credit ;-)

If you would be fine with it, maybe the easiest way would be that instead of having two authors to have just one author with two names. Something like this:
```yaml
roman_martin_manuel_schindler:
  name: Roman Martin, Manuel Schindler
```

That would get both names there, just no link.

Reply:
I liked your approach! Fixed in that way!